ClassificationOfSubgroups/Ch5_PropertiesOfSLOverAlgClosedField/S2_SpecialSubgroups.lean

```diff
@@ -391,87 +369,18 @@ lemma left_subgroupOf_join_right_subgroupOf_join_eq_join_subgroupOf_join {G : Ty
   rw [comap_sup_eq_of_le_range f hH hK]
   simp [f]
 
-lemma simplify₁ : ((D F).subgroupOf (D F ⊔ S F) ⊔ (S F).subgroupOf (D F ⊔ S F)) = ⊤ :=
-  left_subgroupOf_join_right_subgroupOf_join_eq_join_subgroupOf_join (D F) (S F)
-
-/- D and S have trivial interesection, so the following holds -/
-lemma simplify₂ : ((S F).subgroupOf (D F ⊔ S F)).subgroupOf ((D F).subgroupOf (D F ⊔ S F)) = ⊥ := by
-  simp
-  rw [disjoint_iff, ← comap_subtype, ← comap_subtype, ← comap_inf, inf_comm, D_meet_S_eq_bot]
-  simp
-
-/- The second isomorphism theorem -/
-noncomputable def D_join_S_quot_S_subgroupOf_D_join_S_mulEquiv_D_subgroupOf_D_join_S
-  (F : Type*) [Field F] :=
-  (QuotientGroup.quotientInfEquivProdNormalQuotient
-    (H := (D F).subgroupOf (D F ⊔ S F:)) (N := (S F).subgroupOf (D F ⊔ S F :))).symm
-
-#check D_join_S_quot_S_subgroupOf_D_join_S_mulEquiv_D_subgroupOf_D_join_S
-
-def LHS (F : Type*) [Field F] :=
-  @QuotientGroup.equivQuotientSubgroupOfOfEq
-
-
-
-#check QuotientGroup.quotientBot
-
-#check MulEquiv.trans
-
-#check QuotientGroup.equivQuotientSubgroupOfOfEq
-
-#check QuotientGroup.quotientMulEquivOfEq
-
-def RHS (F : Type*) [Field F] :=
```

Comment:
I think these were not used anywhere and felt like an earlier attempts that were superceded by mathlib stuff, but let me know if I was wrong and I will add them back.